cmake/vkfwConfig.cmake

```diff
@@ -0,0 +1,6 @@
+if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.8.0)
+    include(CMakeFindDependencyMacro)
+    find_dependency(glfw3 REQUIRED)
+endif()
```

Comment:
I wonder if different behavior under different cmake versions is a good idea...

That being said, `3.8.0` is an extremely old cmake version. Should we just add a `cmake_minimum_required` statement in here and call it a day instead?